Runtime/Shaders/ComputeShaderHelper.cs

```diff
@@ -29,7 +30,8 @@ internal class ComputeShaderHelper {
             renderTarget.height != maxTextureSize.y) {
             if (renderTarget != null) renderTarget.Release();
 
-            renderTarget = new RenderTexture(maxTextureSize.x, maxTextureSize.y, 32) {
+            // We don't want sRGB here!
+            renderTarget = new RenderTexture(maxTextureSize.x, maxTextureSize.y, 32, RenderTextureFormat.ARGBHalf, RenderTextureReadWrite.Linear) {
```

Comment:
Ok, so you need 16 bit per channel instead of 8 bit as well here.
(Just a comment)